README.md

```diff
@@ -1,448 +1,93 @@
-Several examples of system implementations can be viewed [here][Systems].
+<h2 align="center">
+    <p>A framework for distributed multi-agent reinforcement learning in JAX</p>
+</h2>
+
+<div align="center">
+<a rel="nofollow">
+    <img src="https://img.shields.io/pypi/pyversions/id-mava" alt="Python" />
+</a>
+<a rel="nofollow">
+    <img src="https://badge.fury.io/py/id-mava.svg" alt="PyPi" />
+</a>
+<a rel="nofollow">
+    <img src="https://github.com/instadeepai/Mava/workflows/format_and_test/badge.svg" alt="Formatting" />
+</a>
+<a rel="nofollow">
+    <img src="https://img.shields.io/lgtm/grade/python/g/instadeepai/Mava.svg?logo=lgtm&logoWidth=18" alt="Quality" />
+</a>
+<a rel="nofollow">
+    <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" alt="License" />
+</a>
+<a rel="nofollow">
+    <img src="https://readthedocs.org/projects/id-mava/badge/?version=latest" alt="Docs" />
+</a>
+</div>
 
 <p align="center">
   <img align="center" src="docs/images/animation_quick.gif" width="70%">
 </p>
 
-### Distributed System Training
-
-Mava allows a high level of composability for novel research (i.e. building new systems) as well as making it possible to scale systems in a simple way, using the same underlying multi-agent RL system code. Mava uses [Launchpad][launchpad] for creating distributed programs. In Mava, the system executor (which is responsible for data collection) is distributed across multiple processes each with a copy of the environment. Each process collects and stores data which the Trainer uses to update the parameters of all the actor networks used within each executor. This approach to distributed system training is illustrated on the right in the figure above. Mava also supports distributed training using multiple trainers.
-
-## Getting Started
-
-We have a [Quickstart notebook][quickstart] that can be used to quickly create and train your first Multi-Agent System. For more information on how to use Mava, please view our [usage section](#usage). Finally, for more on Mava's implementation details, please visit our [documentation].
+## Welcome to Mava! 🦁
 
-## Supported Environments
+[**Installation**](#installation-)
+| [**Quickstart**](#quickstart-)
```

Comment:
Very weird, as you said, this works on Github but not locally lol

Reply:
Yes it is weird, but as long as it works on Github, we good ;)